src/assets/styles/fonts/roboto.scss

```diff
@@ -1,56 +1,4 @@
-@font-face {
```

Comment:
I don't think we need to remove all of these. We can reduce the number of font-weights that we support, but we at least need a lighter and bolder variant with italic as well.